### PR TITLE
Process "How to contribute to UI tests"

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | Process "How to contribute to UI tests"
| Fixed ticket | N/A

### CODE REVIEW

**Comment by Progi1984** [comment]
Ping @boubkerbribri & @nesrineabdmouleh

File: content/project-organization/quality-council/processes/contribute-ui-tests.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+### Choose a scenario to automate
+
+_**This part is only available for PrestaShopCorp employees.**_
+* Go to the PrestaShop Forge
+* Access to the Test Repositoy
+* Find a test in the directory `Core` or `Modules` with the state "TO BE AUTOMATED"
+
+### Create the PR
+* The contributor create a PR targetting the development branch for UI tests.
+* The contributor can check that its PR works with the [Github Action `ga.tests.ui.pr`](https://github.com/PrestaShop/ga.tests.ui.pr).
```
Comment (boubkerbribri) [inline content/project-organization/quality-council/processes/contribute-ui-tests.md:34]:
```suggestion
* The contributor can check that his PR works with the [Github Action `ga.tests.ui.pr`](https://github.com/PrestaShop/ga.tests.ui.pr).
```

File: content/project-organization/quality-council/processes/contribute-ui-tests.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+The rules are as follows:
+* The development branch for UI testing is the current development branch.
+* The branch changes as soon as the final release is out.
+
+### Differences between scenarios & tests
+
+Scenarios are the base of UI Tests. They are available on the [PrestaShop Tests Documentation](https://build.prestashop-project.org/test-scenarios/).
+
+Tests are the code after scenarios. They are available on the main repository [in the tests/UI directory](https://github.com/PrestaShop/PrestaShop/tree/develop/tests/UI).
```
Comment (boubkerbribri) [inline content/project-organization/quality-council/processes/contribute-ui-tests.md:21]:
```suggestion
Tests are the code based on scenarios. They are available on the main repository [in the tests/UI directory](https://github.com/PrestaShop/PrestaShop/tree/develop/tests/UI).
```

File: content/project-organization/quality-council/processes/contribute-ui-tests.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* The contributor can check that its PR works with the [Github Action `ga.tests.ui.pr`](https://github.com/PrestaShop/ga.tests.ui.pr).
+
+### Review the PR
+* The review is done by [Software Developers in Tests of the project](/project-organization/people-and-roles/#software-developers-in-test).
+* Some checks are done :
+  * the PR complies with [contribution guidelines](https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/);
+  * the CI is green (lint, typescript check);
+  * the scenario is followed;
+  * the test is executed without errors in the [nightly](https://github.com/PrestaShop/ga.tests.ui.pr);
+  * the test use the Page Object Model pattern.
```
Comment (boubkerbribri) [inline content/project-organization/quality-council/processes/contribute-ui-tests.md:43]:
```suggestion
  * The PR complies with [contribution guidelines](https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/);
  * The CI is green (lint, typescript check);
  * The scenario is followed;
  * The test is executed without errors in the [nightly](https://github.com/PrestaShop/ga.tests.ui.pr);
  * The test use the Page Object Model pattern.
```

**Review by boubkerbribri — APPROVED** [review]

**Pull request merged**